optuna_dashboard/ts/components/App.tsx

```diff
@@ -96,6 +96,15 @@ export const App: FC = () => {
                     />
                   }
                 />
+                <Route
+                  path={URL_PREFIX + "/studies/:studyId/preferenceHistory"}
```

Comment:
Could you avoid using `camelCase` for url path?

```suggestion
                  path={URL_PREFIX + "/studies/:studyId/preference-history"}
```